src/hooks/useSubmissionIdIsValid.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import { ValidatorForm } from 'react-material-ui-form-validator'
+
+const submissionIdInvalidMessage = 'Must be a valid Submission Id'
+
+const useSubmissionIdValidationMessage = (submissionId: string | undefined) => {
+  return React.useMemo(() => {
+    if (!ValidatorForm.getValidator('isUUID', submissionId, true)) {
```

Comment:
As this is a custom validator that we wrote, can we not include this library here and just do the regex check here.

Reply:
Ah, thought it was built in